src/main/java/es/in2/wallet/application/service/impl/UserDataUseCaseServiceImpl.java

```diff
@@ -18,7 +18,7 @@
 public class UserDataUseCaseServiceImpl implements UserDataUseCaseService {
```

Comment:
Necesitaríamos reformular estos nombres de clase. Son un poco verbose :)

Reply:
Changed to DataWorkflow